src/components/Calculator.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please don't disable eslint rules fix them in your code instead you can keep only one which is `/* eslint-disable react/prefer-stateless-function */` because in this project you build your components with classes which you will refactor later in another project.